meson_scripts/init.py

```diff
@@ -218,7 +218,7 @@ def submodule_status(path, sha_commit):
                 cwd=sys.path[0],
             )
             # to update CoolProp external libraries
-        if sha_commit == "0ce42fcf3bb2c373512bc825a4f0c1973a78f307":
+        if "CoolProp" in path:
```

Comment:
ok, this is more clear.

Reply:
please add the new line: **subprocess.run(['git', 'pull', 'origin', 'master'])**
before the line 230: **subprocess.run(['git', 'submodule', 'init'])**

Then SU2 will update CoolProp to the latest version automatically.

You can try on your personal Pc and if it works, you can add this change to this PR.

Reply:
We don't want to simply use the latest version, what if they introduce some change that breaks compatibility?
We use a fixed version for all submodules